packages/browser/src/storage.ts

```diff
@@ -0,0 +1,38 @@
+import { Storage, StorageKey } from '@logto/client';
+import { Nullable } from '@silverhand/essentials';
+
+export const logtoStorageItemKeyPrefix = `logto`;
+
+export class BrowserStorage implements Storage {
+  private readonly storageKey: string;
+
+  constructor(appId: string) {
+    this.storageKey = `${logtoStorageItemKeyPrefix}:${appId}`;
+  }
+
+  getItem(key: StorageKey): Nullable<string> {
+    if (key === 'signInSession') {
+      return sessionStorage.getItem(this.storageKey);
+    }
+
+    return localStorage.getItem(`${this.storageKey}:${key}`);
```

Comment:
may be better? and so do below
```suggestion
    const finalKey = key === 'signInSession'
      ? this.storageKey : `${this.storageKey}:${key}`

    return localStorage.getItem(finalKey);
```

Reply:
`signInSession` is stored in `sessionStorage`.